src/schrodinger/sketcher/molviewer/scene.h

```diff
@@ -99,10 +99,17 @@ class SKETCHER_API Scene : public QGraphicsScene
     QRectF getInteractiveItemsBoundingRect(
         const InteractiveItemFlagType types = InteractiveItemFlag::ALL) const;
 
+    /**
+     * @return the bounding rectangle of all items in the scene. This includes
+     * all interactive items plus any annotation items such as the simplified
+     * stereo label
+     */
+    QRectF getSceneItemsBoundingRect() const;
+
     /**
      * get atoms, bonds, sgroups and non-molecular objects in the scene
-     * @param subset The subset of objects to return (all, selected, selected or
-    hovered).  Defaults to returning all
+     * @param subset The subset of objects to return (all, selected,
+    selected or hovered).  Defaults to returning all
```

Comment:
I realize that you just rewrapped this, but it looks like "selected" appears in this list twice, and line 112 is missing a leading `*`.

Reply:
fixed